core/jazz_acl/components/utils/vault.js

```diff
@@ -185,8 +185,8 @@ function addOrRemoveAllAdminUsersToSafe(environmentDetails, configData, serviceI
 
     for (let key in environments) {
       const environment = environments[key];
-      if (environment.metadata && environment.metadata.SAFE_DETAILS)  //TODO to check the case after the environment changes added
-        processEachPromises.push(processEachPromise(environment.metadata.SAFE_DETAILS.name, configData, serviceId, authToken, adminUsers));
+      if (environment.metadata && environment.metadata.SAFE)  //TODO to check the case after the environment changes added
```

Comment:
should this be `environment.metadata.safe`?

Reply:
While validating with "safe" it was throwing error. So given as "SAFE" and at that time the other PR https://github.com/tmobile/jazz/pull/1211 not developed. So added TODO for  checking the case. Waiting for the Pr to be merged and I will add the case appropriately.